sdk/daml-script/daml3/Daml/Script/Internal/Questions/Submit/Error.daml

```diff
@@ -127,6 +126,9 @@ data SubmitError
     LocalVerdictLockedKeys with
       localVerdictLockedKeys : [AnyContractKey]
           -- ^ Locked contract keys
+  | -- | Upgrade exception
+    UpgradeError with -- We just give the error's prettied message. Scripts doesn't need anything more right now.
```

Comment:
I'll do this in a follow on PR. Rationale is that we need to make some canton side changes first to add `UpgradeErrorType` to `ErrorResource` first (so that we can pass the specific error class through).

Reply:
Oh? I thought you already did [here](https://github.com/DACH-NY/canton/pull/22315/files#diff-3701b4fba9cdd23ec0121c831c9678ee6340dd4e917494862c724ff576802d1fR670)?

Reply:
Exactly. It's the DevErrorType that constrains which classes are considered acceptable. Here I'd done the simple hack and just reused DevErrorType - in the next/follow-on PR I'll fix that.

Reply:
Ah okay, makes sense. Happy to merge then